internal/services/services.go

```diff
@@ -21,31 +21,12 @@ func Lookup(ctx context.Context, kubeClient *kubernetes.Cluster, org, service st
 		return serviceInstance, nil
 	}
 
-	serviceInstance, err = CatalogServiceLookup(ctx, kubeClient, org, service)
-	if err != nil {
-		return nil, err
-	}
-	if serviceInstance != nil {
-		return serviceInstance, nil
-	}
-
 	return nil, errors.New("service not found")
 }
 
 // List returns a ServiceList of all available Services
 func List(ctx context.Context, kubeClient *kubernetes.Cluster, org string) (interfaces.ServiceList, error) {
-
-	customServices, err := CustomServiceList(ctx, kubeClient, org)
-	if err != nil {
-		return nil, err
-	}
-
-	catalogServices, err := CatalogServiceList(ctx, kubeClient, org)
-	if err != nil {
-		return nil, err
-	}
-
-	return append(customServices, catalogServices...), nil
+	return CustomServiceList(ctx, kubeClient, org)
```

Comment:
Tech debt, to be resolved in separate PR:
  - `Merge services.go`, `custom_service.go`, inline custom specific functions into the generic functionality.
  - Drop the service interface ?!

Reply:
I'm wondering if the interface will be useful soon, if we decide to try some other kind of Service (e.g. Crossplane). Usually though, this kind of thinking leads to left-overs in code which nobody ever cleans up. I think I prefer that we removed the interface. Maybe on another PR?

Reply:
See first line of my original comment, another PR is fine.

Reply:
Moved here: #844